netci.groovy

```diff
@@ -32,14 +32,20 @@ def branch = GithubBranchName
                 case 'Windows_NT':
                     newJob.with{
                         steps{
-                            def windowsScript = "call \"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\VsDevCmd.bat\" && cibuild.cmd --target ${runtime}"
+                            // all windows builds do a full framework localized build to produce satellite assemblies
+                            def script = "call \"C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\VsDevCmd.bat\" && cibuild.cmd --target Full --localized-build"
 
-                            // only Desktop support localized builds 
-                            if (runtime == "Desktop") {
-                                windowsScript += " --localized-build"
+                            if (runtime == "Full") {
+                                script += " --scope Test"
+                            }
+                            // .net core builds are localized (they need the satellites from the full framework build), run tests, and also build the nuget packages
+                            else if (runtime == "CoreCLR") {
+                                script == " --scope Compile --build-only"
+                                script += " && cibuild.cmd --target CoreCLR --scope test --localized-Build"
+                                script += " && msbuild build/NugetPackages/CreateNuGetPackages.proj"
```

Comment:
I'd be more comfortable if this was all baked into `cibuild.cmd` so it'd be easier to repro on a local machine.